test.e2e/spec/services.spec.js

```diff
@@ -1251,6 +1251,76 @@ define(['angular', 'given', 'util'], function(angular, given, util) {
       });
     });
 
+    describe('invalid namespaceCommonModels:true', function() {
+      var $injector;
+      before(function() {
+        return given.servicesForLoopBackApp(
+          {
+            models: {
+              Product: {
+                properties: {
+                  name: 'string',
+                  price: { type: 'number' },
+                },
+              },
+            },
+            name: 'lbServices',
+            namespaceCommonModels: true,
+            namespaceDelimiter: '_',
+          })
+          .then(function(createInjector) {
+            $injector = createInjector();
+          });
+      });
+
+      it('should not define the "Auth" common model', function() {
+        expect(function() {
+          $injector.get('Auth');
+        }).to.throw(/Unknown provider/);
+        expect(function() {
```

Comment:
One assert per test please. Create three `it` blocks and put each `expect` into its own test.